### ISSUE TITLE
Sharing Buttons showing blank page when editing "More" buttons with "Official Buttons" selected as Button style.

### ISSUE THREAD

**Issue by danielk-am** [Automattic/wp-calypso#60354]
### Quick summary

https://user-images.githubusercontent.com/92185881/150625951-bacd31ee-5872-411c-9978-5bd21ff57497.mp4

Marketing > Sharing Buttons

Clicking on Edit "More" buttons with the "Official Buttons" option selected for "Button style" leads to a blank page (throws errors in the browser console log, as seen in the other notes section below).

The screen recording shows the expected behavior from the other options selected, with the resulting error from the "Official Buttons" option selected at the end.

Consistent on both Simple and Atomic sites right now, after testing on sites with a Premium, and a Business subscription.

### Steps to reproduce

1. Marketing > Sharing Buttons.
2. In Button style, select "Official Buttons".
3. Click on Edit "More" buttons.
4. Led to a blank page with errors showing up on the browser console logs.

### What you expected to happen

The Edit "More" buttons display the buttons that can be reordered, like how the other options is showing.

### What actually happened

Blank page with errors showing up in the browser console logs.

### Browser, operating system and other notes

Chrome Console errors:

<img width="522" alt="Screenshot 2022-01-22 at 13 02 56" src="https://user-images.githubusercontent.com/92185881/150625914-9e0a9640-a6a2-41c4-bd1e-ddae7c27b217.png">

```
TypeError: Cannot read properties of undefined (reading 'postMessage')
    at Ee.showMorePreview (marketing.3432ba5f2b759cdff70e.min.js:2:60313)
    at Ee.updateMorePreviewVisibility (marketing.3432ba5f2b759cdff70e.min.js:2:60090)
    at Ee.componentDidUpdate (marketing.3432ba5f2b759cdff70e.min.js:2:62401)
    at gi (61606.7ff894d70e590c95786d.min.js:2:191058)
    at Fl (61606.7ff894d70e590c95786d.min.js:2:207806)
    at t.unstable_runWithPriority (61606.7ff894d70e590c95786d.min.js:2:234507)
    at qo (61606.7ff894d70e590c95786d.min.js:2:149926)
    at Nl (61606.7ff894d70e590c95786d.min.js:2:204545)
    at bl (61606.7ff894d70e590c95786d.min.js:2:200656)
    at 61606.7ff894d70e590c95786d.min.js:2:150149
```

### Reproducibility

Consistent

**Comment by danielk-am** [comment]
First encounted in 4709492-zen.

**Comment by ash1eygrace** [comment]
Also, encountered in 4709492-zen. 

Some other findings: 
- Seems to be isolated to the Official buttons, only. On Desktop, (at least in the latest version of Chrome didn't test in other browsers). 
- Works fine in the Mobile App, according to a report in 4709492-zen. 

The workaround I provided was to temporarily switch from Official buttons to other options Text Only, Icon Only, etc. Edit the More Buttons, then switch back to Official buttons.

**Comment by dpasque** [comment]
Bumping to high priority for now, a WSOD is not a good look!

**Comment by vindl** [comment]
@dpasque this doesn't seem to be in the area of FSE or something that Cylon (or Create Focus) maintains. Just wanted to check if the assignment to our team was intentional?

**Comment by dpasque** [comment]
@vindl Hrm, nope, that definitely doesn't look right... So sorry about that, not sure how that happened! This seems to me to fall under Engage and should probably start with Ganon. I've updated it correctly. Thanks for the heads up, and sorry again!

**Comment by creativecoder** [comment]
From the error message, It looks like this might be related to an iframe issue in Calypso, cc @Automattic/team-calypso

**Comment by tyxla** [comment]
Seems like we broke this when we were porting refs to use the modern string syntax in #56719.

The fix is in #62014 - I'd appreciate it if anyone participating in this issue could give it a spin. 🙌
